ui/admin/app/components/form/account/ldap/index.hbs

```diff
@@ -52,27 +52,65 @@
       </F.Error>
     {{/if}}
   </Hds::Form::Textarea::Field>
+  {{#if @model.isNew}}
+    <Hds::Form::TextInput::Field
+      @isRequired={{true}}
+      name='login_name'
+      @value={{@model.login_name}}
+      @isInvalid={{@model.errors.login_name}}
+      disabled={{form.disabled}}
+      {{on 'input' (set-from-event @model 'login_name')}}
+      as |F|
+    >
+      <F.Label>{{t 'form.login_name.label'}}</F.Label>
+      <F.HelperText>{{t 'form.login_name.help'}}</F.HelperText>
+      {{#if @model.errors.login_name}}
+        <F.Error as |E|>
+          {{#each @model.errors.login_name as |error|}}
+            <E.Message>{{error.message}}</E.Message>
+          {{/each}}
+        </F.Error>
+      {{/if}}
+    </Hds::Form::TextInput::Field>
+  {{/if}}
 
-  <Hds::Form::TextInput::Field
-    @isRequired={{true}}
-    name='login_name'
-    @value={{@model.login_name}}
-    @isInvalid={{@model.errors.login_name}}
-    disabled={{form.disabled}}
-    {{on 'input' (set-from-event @model 'login_name')}}
-    as |F|
-  >
-    <F.Label>{{t 'form.login_name.label'}}</F.Label>
-    <F.HelperText>{{t 'form.login_name.help'}}</F.HelperText>
-    {{#if @model.errors.login_name}}
-      <F.Error as |E|>
-        {{#each @model.errors.login_name as |error|}}
-          <E.Message>{{error.message}}</E.Message>
-        {{/each}}
-      </F.Error>
-    {{/if}}
-  </Hds::Form::TextInput::Field>
+  {{#unless @model.isNew}}
+    <Hds::Form::Fieldset as |F|>
 
+      <F.Control>
+        <div class='ldap-section'>
+          <p class='hds-typography-display-300 hds-font-weight-bold'>{{t
+              'form.other_fields.label'
+            }}</p>
+          <Hds::Form::HelperText @controlId='helper-text-for-section'>{{t
+              'form.other_fields.help'
+            }}</Hds::Form::HelperText>
+        </div>
+        <InfoField @value={{@model.login_name}} as |F|>
+          <F.Label>{{t 'form.login_name.label'}}</F.Label>
+        </InfoField>
+        <InfoField @value={{@model.email}} as |F|>
+          <F.Label>{{t 'resources.account.form.email.label'}}</F.Label>
+        </InfoField>
+        <InfoField @value={{@model.full_name}} as |F|>
+          <F.Label>{{t 'resources.account.form.full_name.label'}}</F.Label>
+        </InfoField>
+        <InfoField @value={{@model.dn}} as |F|>
+          <F.Label>{{t 'resources.account.form.dn.label'}}</F.Label>
+        </InfoField>
+        <p class='hds-font-weight-semibold'>{{t
+            'resources.account.form.member_of_groups.label'
+          }}</p>
+        {{#each @model.member_of_groups as |group|}}
+          <Hds::Form::TextInput::Base
+            @value={{group.value}}
+            disabled={{true}}
+            @width='unset'
```

Comment:
<img width="886" alt="image" src="https://github.com/hashicorp/boundary-ui/assets/5783847/94835990-a0a4-472c-b748-53fead2ac341">

Did we want this on one line or each one have its own line? We had a ticket previously filed by product to change each it so each one has its own line for hosts in dynamic host catalogs: https://hashicorp.atlassian.net/browse/ICU-9630.

Any thoughts @joshuaogle?

Reply:
I like the look of it in columns but for longer group names and consistency having 1 per line is best 👍

Reply:
ty for the feedback, changed to rows!